src/components/common/settings.tsx

```diff
@@ -0,0 +1,144 @@
+import { YellowButton } from "@/components/common/button";
+import { Input } from "@/components/common/input";
+import Modal from "@/components/common/modal";
+import { SelectDark } from "@/components/common/select";
+import GgxContext from "@/components/providers/ggx";
+import { errorHandler } from "@/services/api";
+import GGXWallet, { type Account } from "@/services/ggx";
+import { type ChangeEvent, useContext, useEffect, useState } from "react";
+type Radion = "" | "Node" | "Mainnet" | "Testnet";
+const NODE_PROVIDERS: Radion[] = ["Node", "Mainnet", "Testnet"];
```

Comment:
options must be: 
1. env.NEXT_PUBLIC_PARACHAIN_URL
2. manual input

Reply:
its only prototype but thanks for review will update it